run_gcs_to_bq/Dockerfile

```diff
@@ -15,7 +15,7 @@ COPY ./python ./python
 # Install production dependencies.
 RUN pip install -r run_gcs_to_bq/requirements.txt
 # Install shared packages
-RUN pip install ./python/ingestion
+RUN pip install ./python
```

Comment:
What was the motivation for changing this to install everything in `python/`?

Reply:
With find_packages() we could install all the required packages and only copy over the packages we need. See comment above